### DISCUSSION TITLE
Linting/Contributing Guide

### DISCUSSION THREAD

**Discussion by robbievanleeuwen** [robbievanleeuwen/section-properties#52] [General]
Following on from the already merged PR #47, here follows discussion relating to incorporating `black` linting, CI and contributing guides.

**Comment by Spectre5** [comment]
Great, glad to see these discussions started!  Ill start work on this, but maybe not in earnest until the shapely stuff gets merged.

**Reply by robbievanleeuwen** [reply]
Agree that it's probably best to wait until we get the shapely merge.
